### ISSUE TITLE
Compute area of a PolyLine?

### ISSUE THREAD

**Issue by drewish** [cinder/Cinder#679]
I needed to compare the area of some shapes defined using PolyLines. I didn't see an obvious way to compute the area so I wrote this function up: 
```
// Assumes line does not self intersecting
// Treats the line as closed (you wanted area right?)
// http://www.mathsisfun.com/geometry/area-irregular-polygons.html
float area( PolyLine2f line ) {
	float sum = 0.0;
	if ( line.size() > 1 ) {
		PolyLine2f::iterator it;
		Vec2f prev, curr;
		for ( it = line.begin(), prev = *it++; it != line.end() ; it++ ) {
			curr = *it;
			sum += ( prev.y + curr.y ) / 2.0 * ( curr.x - prev.x );
			prev = curr;
		}
	}
	return sum;
}
```
https://gist.github.com/drewish/d5d95cdf398e5f7e7240

If there's any interested I'd be happy to try to put together a PR to clean this up and add it as method of the class

**Comment by drewish** [comment]
I guess boost::geometry has an area function. Probably better to just expose that.

**Comment by drewish** [comment]
Here's the forum topic I opened for this PR: https://forum.libcinder.org/topic/exposing-more-boost-geometry-functionality-in-polyline

**Comment by andrewfb** [comment]
Thanks for this - should be addressed by ccdb941e0b2bc25b6bca89a0fb7582e4d5ae1707

**Comment by drewish** [comment]
Awesome
